### ISSUE TITLE
AttributeError: module 'xarray.core.formatting_html' has no attribute 'CSS_STYLE'

### ISSUE THREAD

**Issue by Claes1981** [kiudee/chess-tuning-tools#101]
* Chess Tuning Tools version: 0.6.0
* Python version: 3.8.5
* Operating System: OpenSUSE Tumbleweed

### Description

Crash after updating some packages to latest versions in virtual environment.

### What I Did

```
tune local --tuning-config Stockfish_20080713_bmi2_pgo_S200728-1442_b_config.json --data-path Stockfish_20080713_bmi2_pgo_S200728-1442_b.npz --logfile Stockfish_20080713_bmi2_pgo_S200728-1442_b.log --plot-path Stockfish_20080713_bmi2_pgo_S200728-1442_b --verbose --confidence 0.8 --gp-initial-burnin 200 --gp-burnin 100 --plot-every 5
Traceback (most recent call last):
  File "/home/claes/Chess-tuning-tools/env/bin/tune", line 5, in <module>
    from tune.cli import cli
  File "/home/claes/Chess-tuning-tools/env/lib64/python3.8/site-packages/tune/cli.py", line 13, in <module>
    from bask.optimizer import Optimizer
  File "/home/claes/Chess-tuning-tools/env/lib64/python3.8/site-packages/bask/__init__.py", line 10, in <module>
    from .optimizer import Optimizer
  File "/home/claes/Chess-tuning-tools/env/lib64/python3.8/site-packages/bask/optimizer.py", line 4, in <module>
    from arviz import hdi
  File "/home/claes/Chess-tuning-tools/env/lib64/python3.8/site-packages/arviz/__init__.py", line 28, in <module>
    from .data import *
  File "/home/claes/Chess-tuning-tools/env/lib64/python3.8/site-packages/arviz/data/__init__.py", line 2, in <module>
    from .inference_data import InferenceData, concat
  File "/home/claes/Chess-tuning-tools/env/lib64/python3.8/site-packages/arviz/data/inference_data.py", line 15, in <module>
    from ..utils import _subset_list, HtmlTemplate
  File "/home/claes/Chess-tuning-tools/env/lib64/python3.8/site-packages/arviz/utils.py", line 670, in <module>
    class HtmlTemplate:
  File "/home/claes/Chess-tuning-tools/env/lib64/python3.8/site-packages/arviz/utils.py", line 696, in HtmlTemplate
    css_template = f"<style> {xr.core.formatting_html.CSS_STYLE}{specific_style} </style>"
AttributeError: module 'xarray.core.formatting_html' has no attribute 'CSS_STYLE'

```
Virtual environment packages versions:
```
 pip freeze
arviz==0.9.0
astroid==2.4.2
atomicwrites==1.4.0
bask==0.10.0
certifi==2020.6.20
cftime==1.2.1
chess-tuning-tools==0.6.0
click==7.1.2
cycler==0.10.0
emcee==3.0.2
isort==5.5.2
joblib==0.16.0
kiwisolver==1.2.0
lazy-object-proxy==1.4.3
matplotlib==3.3.2
mccabe==0.6.1
netCDF4==1.5.4
numpy==1.19.2
packaging==20.4
pandas==1.1.2
Pillow==7.2.0
pip-review==1.1.0
psycopg2==2.8.6
pyaml==20.4.0
pylint==2.6.0
pyparsing==2.4.7
python-dateutil==2.8.1
pytz==2020.1
PyYAML==5.3.1
scikit-learn==0.23.2
scikit-optimize==0.8.1
scipy==1.5.2
six==1.15.0
SQLAlchemy==1.3.19
threadpoolctl==2.1.0
toml==0.10.1
tqdm==4.49.0
wrapt==1.12.1
xarray==0.16.1
```

**Comment by Claes1981** [comment]
Downgrading xarray to 0.16.0 again seems to solve the problem.

**Comment by kiudee** [comment]
@Claes1981 interesting, xarray 0.16.1 was released an hour ago. After I did all my tests.

Looks like arviz depends on CSS_STYLE being there, which was removed here:
https://github.com/pydata/xarray/pull/4297
I will open an upstream issue.

**Comment by kiudee** [comment]
Notified the devs here:
https://github.com/arviz-devs/arviz/issues/1387

**Comment by Claes1981** [comment]
Thanks, no problem. The fix was easy when I discovered that it was just to downgrade xarray. :)

**Comment by kiudee** [comment]
Since arviz 0.10.0 was now released, which fixes this issue, I also released a hotfix to bask (0.10.1). Updating both should also now fix this problem.
